.github/workflows/redpanda-migration-build.yml

```diff
@@ -18,8 +18,7 @@ on:
       - 'v*'
     paths:
       - .github/workflows/redpanda-migration-build.yml
-      - 'src/go/cluster-to-redpanda-migration/**'
-      - 'src/go/k8s/**'
+      - src/go/cluster-to-redpanda-migration
```

Comment:
this will undo the fix to the path. It'll be better to leave this change out and only leave the removal of the workflow

Reply:
oops, thanks for catching that.

Reply:
Not leaving out the change entirely because there's no reason to test the migration tool when there are changes to src/go/k8s.